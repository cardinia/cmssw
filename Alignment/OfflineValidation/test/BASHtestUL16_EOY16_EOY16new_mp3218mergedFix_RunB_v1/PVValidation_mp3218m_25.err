27-Sep-2019 21:38:12 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/2EAB79A5-2A0B-E911-B196-6CC2173BBF20.root
27-Sep-2019 21:38:15 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/2EAB79A5-2A0B-E911-B196-6CC2173BBF20.root
Begin processing the 1st record. Run 272815, Event 3854894, LumiSection 6 on stream 0 at 27-Sep-2019 21:38:22.471 CEST
Begin processing the 1001st record. Run 272815, Event 8828478, LumiSection 10 on stream 0 at 27-Sep-2019 21:38:25.428 CEST
Begin processing the 2001st record. Run 272815, Event 15791468, LumiSection 15 on stream 0 at 27-Sep-2019 21:38:26.665 CEST
Begin processing the 3001st record. Run 272816, Event 2305538, LumiSection 2 on stream 0 at 27-Sep-2019 21:38:29.027 CEST
Begin processing the 4001st record. Run 272816, Event 7462860, LumiSection 6 on stream 0 at 27-Sep-2019 21:38:30.852 CEST
Begin processing the 5001st record. Run 272816, Event 15875294, LumiSection 11 on stream 0 at 27-Sep-2019 21:38:33.171 CEST
Begin processing the 6001st record. Run 272816, Event 20993974, LumiSection 15 on stream 0 at 27-Sep-2019 21:38:35.484 CEST
Begin processing the 7001st record. Run 272816, Event 28964525, LumiSection 20 on stream 0 at 27-Sep-2019 21:38:37.073 CEST
Begin processing the 8001st record. Run 272828, Event 452105, LumiSection 2 on stream 0 at 27-Sep-2019 21:38:39.241 CEST
Begin processing the 9001st record. Run 272828, Event 10492725, LumiSection 21 on stream 0 at 27-Sep-2019 21:38:41.188 CEST
Begin processing the 10001st record. Run 272828, Event 21528518, LumiSection 43 on stream 0 at 27-Sep-2019 21:38:42.439 CEST
Begin processing the 11001st record. Run 272936, Event 270391986, LumiSection 189 on stream 0 at 27-Sep-2019 21:38:44.304 CEST
Begin processing the 12001st record. Run 272936, Event 277362128, LumiSection 194 on stream 0 at 27-Sep-2019 21:38:46.432 CEST
Begin processing the 13001st record. Run 272936, Event 285938590, LumiSection 200 on stream 0 at 27-Sep-2019 21:38:48.786 CEST
Begin processing the 14001st record. Run 272936, Event 295520768, LumiSection 206 on stream 0 at 27-Sep-2019 21:38:50.974 CEST
27-Sep-2019 21:38:52 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver1-v1/90000/2EAB79A5-2A0B-E911-B196-6CC2173BBF20.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[2.814MB/2.814MB][100%][==================================================][2.814MB/s]  [2.814MB/2.814MB][100%][==================================================][2.814MB/s]  
