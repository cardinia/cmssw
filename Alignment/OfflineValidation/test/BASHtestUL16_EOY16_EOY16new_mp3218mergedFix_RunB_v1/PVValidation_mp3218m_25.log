000 (3196546.025.000) 09/27 21:31:17 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196546.025.000) 09/27 21:38:01 Started transferring input files
	Transferring to host: <188.184.138.186:9618?addrs=188.184.138.186-9618+[2001-1458-301-45--100-75]-9618&noUDP&sock=31279_a291_130813>
...
040 (3196546.025.000) 09/27 21:38:01 Finished transferring input files
...
001 (3196546.025.000) 09/27 21:38:01 Job executing on host: <188.184.138.186:9618?addrs=188.184.138.186-9618+[2001-1458-301-45--100-75]-9618&noUDP&sock=5594_98e1_3>
...
006 (3196546.025.000) 09/27 21:38:10 Image size of job updated: 209072
	114  -  MemoryUsage of job (MB)
	116452  -  ResidentSetSize of job (KB)
...
006 (3196546.025.000) 09/27 21:38:55 Image size of job updated: 209072
	114  -  MemoryUsage of job (MB)
	116456  -  ResidentSetSize of job (KB)
...
040 (3196546.025.000) 09/27 21:38:55 Started transferring output files
...
040 (3196546.025.000) 09/27 21:38:55 Finished transferring output files
...
005 (3196546.025.000) 09/27 21:38:55 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:32, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:32, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	4541  -  Run Bytes Sent By Job
	1021  -  Run Bytes Received By Job
	4541  -  Total Bytes Sent By Job
	1021  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     2925        1   2644415 
	   Memory (MB)          :      114     2000      2000 
...
