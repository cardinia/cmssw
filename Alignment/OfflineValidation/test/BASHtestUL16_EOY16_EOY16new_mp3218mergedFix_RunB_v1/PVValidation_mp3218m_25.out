Job started at  Fri Sep 27 21:38:02 CEST 2019
batch dir: /pool/condor/dir_7166 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_272814_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 2.9M -rw-r--r--. 1 tkello zh 2.9M Sep 27 21:38 PVValidation_mp3218m_272814.root -rw-r--r--. 1 tkello zh 20K Sep 27 21:38 PVValidation_mp3218m_272814_cfg.py -rw-r--r--. 1 tkello zh 3.3K Sep 27 21:38 _condor_stderr -rw-r--r--. 1 tkello zh 521 Sep 27 21:38 _condor_stdout -rwxr-xr-x. 1 tkello zh 1021 Sep 27 21:38 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:38 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:38 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:38 var
Job ended at  Fri Sep 27 21:38:55 CEST 2019
